### ISSUE TITLE
The selected folder wasn't an installation directory of wow. Please select the wow folder

### ISSUE THREAD

**Issue by Vysthril** [ephraim/lcurse#62]
Hello.  I am very new to Linux and I am running into an issue trying to get this to recognize my WoW installation folder.

Installation Path:  /home/user/Games/world-of-warcraft/drive_c/Program Files (x86)/World of Warcraft/

Resulting Error:  The selected folder wasn't an installation directory of wow.
Please select the wow folder

I am sure it is something simple (such as the linking mentioned in the Read Me) however with all attempts to set up linking and get it to accept this folder as the actual WoW installation folder; I have failed.

Help is greatly appreciated and thank you.

**Comment by ilu33** [comment]
You sure you've got a 32bit install? For 64bit WoW the folder should be 
/home/user/......./drive_c/Program Files/World of Warcraft
And note that you should not add a slash at the end. At least for me it's working without the slash.

**Comment by Vysthril** [comment]
Thank you!  I discovered my issue yesterday.  Being new to Linux - I knew that spaces " " were an issue and required quotes or a \, but I was unaware parentheses ( ) were subject to the same issue.  Once I updated the link correctly (as the program itself didn't update the spaces and parentheses, even when I selected the folder through it) - it recognized the folder.  The other option I found was moving the program directly into the install folder fixed it.

thanks again for the response and hopefully this will help the next person trying to get it to work
